core/app/assets/js/ui/molecules/inline-link.js

```diff
@@ -28,16 +28,21 @@ export default function InlineLink( props ) {
 				</Link>
 			</LocationProvider>
 		),
-		getExternalLink = () => (
+		getExternalLink = () => {
+		return (
 			<a
 				href={ props.url }
 				target={ props.target }
 				rel={ props.rel }
 				className={ className }
+				onClick={ () => {
+					return 'undefined' !== typeof ( props.eventTrack ) ? props.eventTrack( props.trackingParams?.trackName, props.trackingParams?.event, props.trackingParams?.details ) : null;
```

Comment:
```suggestion
					
				props.eventTrack.?( 
					props.trackingParams.trackName, 
					props.trackingParams.event, 
					props.trackingParams.details 
				);
```

Why do you need a return?

Reply:
add PropTypes